api/nodes/serializers.py

```diff
@@ -1016,21 +1068,6 @@ def get_related_url(self, obj):
     class Meta:
         type_ = 'institutions'
 
-    def get_institutions_to_add_remove(self, institutions, new_institutions):
-        diff = relationship_diff(
-            current_items={inst._id: inst for inst in institutions.all()},
-            new_items={inst['_id']: inst for inst in new_institutions}
-        )
-
-        insts_to_add = []
-        for inst_id in diff['add']:
-            inst = Institution.load(inst_id)
-            if not inst:
-                raise exceptions.NotFound(detail='Institution with id "{}" was not found'.format(inst_id))
```

Comment:
@sloria Just for the record, those items weren't mine, I was just moving them from here.